src/Sarif/Visitors/SarifTransformerUtilities.cs

```diff
@@ -13,6 +15,14 @@ namespace Microsoft.CodeAnalysis.Sarif.Visitors
 {
     public static class SarifTransformerUtilities
     {
+        private static readonly Regex CommitShaRegex = new Regex(@"^(?<hash>[0-9a-f]{40}).*$", RegexOptions.Compiled);
+        private static readonly int CommitShaLength = 40;
+        private static readonly Regex AuthorTZRegex = new Regex(@"^author-tz", RegexOptions.Compiled);
+        private static readonly Regex AuthorTimeRegex = new Regex(@"^author-time", RegexOptions.Compiled);
+        private static readonly Regex AuthorMailRegex = new Regex(@"^author-mail", RegexOptions.Compiled);
+        private static readonly string authorMailString = "author-mail <>";
+        private static readonly Regex AuthorRegex = new Regex(@"^author", RegexOptions.Compiled);
```

Comment:
Can you use the snippet above:
```csharp

        private static readonly int CommitShaLength = 40;
        private static readonly string authorMailString = "author-mail <>";
        private static readonly Regex AuthorRegex = new Regex(@"^author", RegexOptions.Compiled);
        private static readonly Regex AuthorTZRegex = new Regex(@"^author-tz", RegexOptions.Compiled);
        private static readonly Regex AuthorTimeRegex = new Regex(@"^author-time", RegexOptions.Compiled);
        private static readonly Regex AuthorMailRegex = new Regex(@"^author-mail", RegexOptions.Compiled);
        private static readonly Regex CommitShaRegex = new Regex(@"^(?<hash>[0-9a-f]{40}).*$", RegexOptions.Compiled);

```

it is ordered by textstring, so, easier to ready

Reply:
Makes sense. Fixed.